core/renderers/common/block_rendering.js

```diff
@@ -33,33 +33,6 @@ Blockly.blockRendering.rendererMap_ = {};
  */
 Blockly.blockRendering.useDebugger = false;
 
-/**
- * Registers a new renderer.
- * @param {string} name The name of the renderer.
- * @param {!Function} rendererClass The new renderer class
- *     to register.
- * @throws {Error} if a renderer with the same name has already been registered.
- */
-Blockly.blockRendering.register = function(name, rendererClass) {
```

Comment:
Don't remove, deprecate and internally use the new registry.

Reply:
done